graylog2-server/src/main/java/org/graylog/events/processor/aggregation/AggregationKeyResult.java

```diff
@@ -35,14 +35,16 @@ public abstract class AggregationKeyResult {
 
     public abstract ImmutableList<AggregationSeriesValue> seriesValues();
 
+    public abstract Optional<DateTime> from();
```

Comment:
Hmm, I don't see how and where we transfer this to the actual Event

Reply:
We don't :/ Thought I had set the interval, but I was only copying the timestamp from the keyresult.